### ISSUE TITLE
Improve monitoring of pulls website

### ISSUE THREAD

**Issue by mattl** [w3c/wptdash#43]
Restart things when they're down, etc.

**Comment by foolip** [comment]
@mdittmer already has some monitoring at https://bit.ly/ecosystem-infra-status, but we removed ecosystem-infra from notifications because of noise.

**Comment by mattl** [comment]
Oh yeah, and I have my own monitoring too, but it just tells me when its down. I'd like to improve that to actually do something.

**Comment by foolip** [comment]
Oh, so responding with 504 counts as being up? I have something very rudimentary in https://github.com/whatwg/misc-server/blob/master/test/wiki.js, but I guess you'd want something that runs more than every 24 hours :)

**Comment by mattl** [comment]
No, it doesn't count as being up. Ideally, when that happens something can restart the service after a few retries. This helps in the situation where things are down at 3am more than when things are down at 3pm on a weekday.
